packages/lesswrong/components/tagging/TagHoverPreview.tsx

```diff
@@ -42,28 +42,32 @@ const TagHoverPreview = ({
   // element ID in the dom) eg: "Further_reading"
   const hashId = hash.slice(1);
 
-  const {tag} = useTagPreview(slug, hashId, {skip: noPrefetch});
-  const { showPostCount: showPostCountQuery, useTagName: useTagNameQuery } = targetLocation.query
+  const {tag, loading} = useTagPreview(slug, hashId, {skip: noPrefetch});
+  const { showPostCount: showPostCountQuery, useTagName: useTagNameQuery, lens: lensQuery } = targetLocation.query
   const showPostCount = showPostCountQuery === "true" // query parameters are strings
   const tagName = useTagNameQuery === "true" ? tag?.name : undefined // query parameters are strings
-
+  const previewSlug = lensQuery ?? slug;
   // Remove showPostCount and useTagName query parameters from the link, if present
   const linkTarget = normalizeTagLink(href);
 
+  const isRedLink = (!tag && !noPrefetch && !loading) || tag?.isPlaceholderPage;
```

Comment:
FYI: this will cause broken tag links to turn red.  IMO this seems worth keeping for the EA forum, though you may want to check the color/styling.